### DISCUSSION TITLE
m306-projektmanagement-p2

### DISCUSSION THREAD

**Discussion by mxrni** [mxrni/marni-markdown#3] [General]
comment section for m306-projektmanagement-p2

**Comment by janvetsch** [comment]
Chan mir jemand Top-Down und Bottom-Up churz erklära? und villicht no wieso das die so heissend?
Dankeee

**Reply by nopoxy** [reply]
Top-Down = obä-abä (s höö isch jetz diunna)
Bottom-Up = unä-ufä(s höö isch jetz dioba)

**Reply by jubr001** [reply]
has ned checkt vorher aber dank diar jetzt scho🙌

**Reply by janvetsch** [reply]
Danke für die baba antwort kuss diggi

**Comment by svedana** [comment]
Florian ka das voll set er
